Author: Michael R. Crusoe <crusoe@debian.org>
Description: Skip jemalloc, it isn't used by the only upstream of rust-rocksdb (sourmash)
Forwarded: not-needed
--- rocksdb.orig/Cargo.toml
+++ rocksdb/Cargo.toml
@@ -73,7 +73,6 @@
     "bzip2",
 ]
 io-uring = ["librocksdb-sys/io-uring"]
-jemalloc = ["librocksdb-sys/jemalloc"]
 lz4 = ["librocksdb-sys/lz4"]
 multi-threaded-cf = []
 rtti = ["librocksdb-sys/rtti"]
--- rocksdb.orig/Cargo.toml.orig
+++ rocksdb/Cargo.toml.orig
@@ -22,7 +22,6 @@
 
 [features]
 default = ["snappy", "lz4", "zstd", "zlib", "bzip2"]
-jemalloc = ["librocksdb-sys/jemalloc"]
 io-uring = ["librocksdb-sys/io-uring"]
 valgrind = []
 snappy = ["librocksdb-sys/snappy"]
